hs_core/views/user_rest_api.py

```diff
@@ -1,30 +1,53 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
-from theme.models import UserProfile
+from theme.forms import UserProfileForm
 
 
 class UserInfo(APIView):
     def get(self, request):
         if not request.user.is_authenticated():
             return Response({ "title": "None", "organization": "None"})
 
-        user_info = {"username": request.user.username}
+        user_info = {
+            "username": ['Username', request.user.username],
+            "email": ['Email', ''],
+            "id": ['Id',''],
+            "first_name": ['First name', ''],
+            "last_name": ['Last name', ''],
+            "middle_name": [UserProfileForm.base_fields['middle_name'].label, ''],
+            "state": [UserProfileForm.base_fields['state'].label, ''],
+            "zipcode": [UserProfileForm.base_fields['zipcode'].label, ''],
+            "country": [UserProfileForm.base_fields['country'].label, ''],
+            "date_of_birth": [UserProfileForm.base_fields['date_of_birth'].label, ''],
+            "last_four_ss": [UserProfileForm.base_fields['last_four_ss'].label, ''],
+            "phone_1": [UserProfileForm.base_fields['phone_1'].label, '']
+        }
 
         if request.user.email:
-            user_info['email'] = request.user.email
+            user_info['email'][1] = request.user.email
         if request.user.first_name:
-            user_info['first_name'] = request.user.first_name
+            user_info['first_name'][1] = request.user.first_name
         if request.user.id:
-            user_info['id'] = request.user.id
+            user_info['id'][1] = request.user.id
         if request.user.last_name:
```

Comment:
Rather than having these if-blocks here you can set the value the `user_info[*][1]` directly in its definition using `getattr`. Like...

```python
user_info = {
    "username": ['Username', request.user.username],
    "email": [str(UserChangeForm.base_fields['email'].label), getattr(request.user, 'email', ''),
    ...
    ...
}
```
```